namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-backend-staging/resources/rds.tf

```diff
@@ -0,0 +1,60 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+
+# IMP NOTE: Updating to module version 5.3, existing database password will be rotated.
+# Make sure you restart your pods which use this RDS secret to avoid any down time.
+
+module "cla_backend_rds" {
+  source               = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.3"
+  cluster_name         = var.cluster_name
+  cluster_state_bucket = var.cluster_state_bucket
+  team_name            = var.team_name
+  business-unit        = var.business-unit
+  application          = var.application
+  is-production        = var.is-production
+
+  # enable performance insights
+  performance_insights_enabled = true
+
+  db_name = "cla_backend"
+  # change the postgres version as you see fit.
+  db_engine_version      = "9.6"
+  environment-name       = var.environment-name
+  infrastructure-support = var.infrastructure-support
+
+  # rds_family should be one of: postgres9.4, postgres9.5, postgres9.6, postgres10, postgres11
+  # Pick the one that defines the postgres version the best
+  rds_family = "postgres96"
```

Comment:
This needs to be `postgres9.6` with a dot between 9 and 6

the plan pipeline check doesn't pick this up, but it will fail when terraform actually makes calls to the AWS API.